aws/resource_aws_acm_certificate_test.go

```diff
@@ -539,20 +539,30 @@ func TestAccAWSAcmCertificate_imported_DomainName(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "tags.%", "0"),
 					resource.TestCheckResourceAttr(resourceName, "domain_name", "example.com"),
 				),
+				ExpectNonEmptyPlan: true, // The certificate body is regenerated every time
+			},
+			{
+				Config: testAccAcmCertificateConfig_selfSigned("example"),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, "tags.%", "0"),
+					resource.TestCheckResourceAttr(resourceName, "domain_name", "example.com"),
+				),
+				ExpectNonEmptyPlan: true, // The certificate body is regenerated every time
```

Comment:
The `tls_locally_signed_cert` is regenerated every time because it's a new request it seems